onnxruntime/core/providers/cpu/ml/tree_ensemble_common.h

```diff
@@ -35,52 +38,60 @@ class TreeEnsembleCommon {
   TreeEnsembleCommon(int parallel_tree,
                      int parallel_N,
                      const std::string& aggregate_function,
-                     const std::vector<OTYPE>& base_values,
+                     const std::vector<float>& base_values,
```

Comment:
>const std::vector<float> [](http://example.com/codeflow?start=21&length=24)

According to the most recent changes in the coding standards, we prefer to pass gsl::span<const T>. This way it is container independent during refactoring and we can use either std::vector, InlinedVector or any other type of contiguous memory.

Reply:
I suggest doing that in another PR.